docker/node-modules-clean.sh

```diff
@@ -0,0 +1,4 @@
+rm -rf node_modules/rxjs/src/
+rm -rf node_modules/rxjs/bundles/
+rm -rf node_modules/rxjs/_esm5/
+rm -rf node_modules/rxjs/_esm2015/
```

Comment:
Why cleaning these specific node_modules? Maybe the answer is worth a comment either in this script or in the Dockerfile

Reply:
hmmm. not sure about that.  replicated them from `iam-cache-server` as the org-creator service also uses those packages, I thought it would be a good idea to have them here as well. I'll probably just remove them from here since I am not entirely sure what  they do. what do you think?

Reply:
Maybe it is to reduce docker image size? Seems plausible, that's what is being done in this article, step 3.4: https://medium.com/trendyol-tech/how-we-reduce-node-docker-image-size-in-3-steps-ff2762b51d5a (just did a quick google for "remove rxjs docker" 😆 ).
Maybe we assume that it's to reduce image size and we keep the script and add a comment that this is our understanding?